app/main.go

```diff
@@ -352,8 +346,14 @@ type bufioSyncer struct{ *bufio.Writer }
 
 func (b *bufioSyncer) Sync() error { return b.Flush() }
 
-func fatalIfError(logger *ui.UI, err error) {
+func fatalIfError(logger *ui.UI, ctx *kong.Context, err error) {
 	if err != nil {
 		logger.Task("hermit").Fatalf("%s", err)
```

Comment:
I still use `Fatalf` instead of `Errorf` because they will output different log colours -- `Errorf` is yellow and `Fatalf` is red